tests/test_cmdbase.py

```diff
@@ -87,3 +87,33 @@ class TestClass(BaseCommand):
     tc = TestClass("group", "config")
     with pytest.raises(NotImplementedError):
         tc.run([])
+
+
+# -- tests for strings in commands
+
+
+@pytest.mark.parametrize("command", all_commands)
+def test_aesthetic_help_msg(command):
+    """All the real commands help msg start with uppercase and doesn't end with a dot."""
+    msg = command.help_msg
+    assert msg[0].isupper() and msg[-1] != "."
+
+
+@pytest.mark.parametrize("command", all_commands)
+def test_aesthetic_args_options_msg(command, config):
+    """All the real commands args help messages start with uppercase and dont' end with a dot."""
```

Comment:
This looks more like an integration test.

```suggestion
    """All real command args help messages start with uppercase and do not end with a dot."""
```